src/azure/cli/application.py

```diff
@@ -125,8 +125,6 @@ def execute(self, argv):
         self.raise_event(self.TRANSFORM_RESULT, event_data=event_data)
         self.raise_event(self.FILTER_RESULT, event_data=event_data)
         return CommandResultItem(event_data['result'],
-                                 simple_output_query=
-                                 command_table[args.command].simple_output_query,
```

Comment:
>command_table[args.command].simple_output_query [](start = 33, length = 47)

Will you have a separate PR to remove all existing simple_output_queries or are you going to do that as an iteration on this PR?